### DISCUSSION TITLE
1984 개발일정 23-04 2week

### DISCUSSION THREAD

**Discussion by fkdl0048** [The-Day-In-1984/1984_Project#96] [General]
# 각 파트 별 개발일정  

<img src="https://user-images.githubusercontent.com/84510455/230813755-5daacee5-c6a1-4141-a88e-3ab5624e015a.gif" width="200" height="200"/>

**Comment by fkdl0048** [comment]
## 시험 기간 이슈로 휴식..

*필요하거나 간단한 개발들은 언제나 환영..*

**Comment by aminseo1023** [comment]
@kyoonge @ccmuk58 @fkdl0048 @dkssu46 
플밍에게 질문…! 혹시 플밍 본격적으로 시작하려면 기획 쪽에서 해줘야 하는 젤 급한 게 뭘까?? 급한 것 위주로 일정을 짜려고 해

**Reply by ccmuk58** [reply]
난 아직 크게 없는 듯,,?
작업 해야 되는데,.,, 😿

**Reply by fkdl0048** [reply]
1. 각 기능별로 조금 구체화(레퍼런스 위주로)
3. 게임 전체적인 시스템

**Reply by kyoonge** [reply]
어어 나도 게임화면(플랫포머)이 어떤 모습일지 대충 파악할 수 있도록 레퍼런스가 있었으면 좋겠어...!
캐릭터 조작도 세부적인건 함께 얘기 나눠봐야 할 것 같엉,,

**Comment by ccmuk58** [comment]
- #74 

시험 기간 4월 말이라 이번 주는 위 사항 진행 예정

++
- #93 
지난 작업 개선하는게 생각보다 오래 걸렸어,,, 
4월 3주차에 이어서 할게 (#74)

**Comment by kyoonge** [comment]
- #94 

여기 코드 수정할게요!

**Comment by kangjieum** [comment]
나도 시험기간이긴 한데 이번주보다 다음주가 더 바쁠 것 같아서 ..
아예 이번주에 사임 1차 컨셉시안 짜서 4월 17일(월) 까지 올려놓을게~!
대신 다음주는 쉬어갈게요..

+) 2차 시안까지 나온 캐릭터들은 (사임 제외 5명) 다다음주에 수정본 올라갈 예정.. !

**Comment by aminseo1023** [comment]
저는 플랫포머 맵 레퍼런스 찾아서 공유하고, 기획 공통 작업으로 미션 짜기 하겠습니다~!

**Comment by dkssu46** [comment]
- #69 
미션 레퍼런스 빠져있던거 다 붙여넣고, 기획 공통으로 미션 짜기 하면서
레퍼런스로 쓸 미션 하나 이슈에 올려놓을게요

**Comment by sueyeoun** [comment]
정신없어서 늦었슴다! 
시험 기간 이유로 휴식하겠습니다
